api-spec/extensions/query/README.md

```diff
@@ -17,7 +17,7 @@ The syntax for the `query` filter is:
 }
 ```
 
-Each property to search is an entry in the `query` filter. <operator> can be one of: `eq`, `neq`, `lt`, `lte`, `gt`, `gte`, `startsWith`, `endsWith`, `contains`. 
+Each property to search is an entry in the `query` filter. <operator> can be one of: `eq`, `neq`, `lt`, `lte`, `gt`, `gte`, `startsWith`, `endsWith`, `contains`, `in`. 
```

Comment:
i forgot to add `in` to this list when I originally added it in the yaml and example below.